spec/factories/courses.rb

```diff
@@ -1,8 +1,9 @@
 FactoryGirl.define do
   factory :course do
-    department "EECS"
-    course_number 132
-    title "Intro to Java"
+    department {Forgery(:basic).text[0...4].upcase}
+    course_number {Forgery(:basic).number*100}
```

Comment:
Surrounding space missing for operator '*'.<br>Space missing inside {.<br>Space missing inside }.